src/cuda/utils.cpp

```diff
@@ -25,7 +25,14 @@ CUFINUFFT_BIGINT next235beven(CUFINUFFT_BIGINT n, CUFINUFFT_BIGINT b)
 
 // ----------------------- helpers for timing (always stay double prec)...
 
-void CNTime::start() { gettimeofday(&initial, 0); }
+void CNTime::start() {
```

Comment:
The cpu library is already windows compatible in this case. 

I suggest moving the CPU utils in a global folder that can be included by both (cpu and cuda) and using that. We could wrap all that functionality in a library that is linked by both as the reason for this duplication is that cuFINUFFT was in a separate repo before.

Reply:
Would you mind putting the timer implementation in the util header of finufft and using it directly in cufinufft?

Reply:
I'd rather move to a separate lib. Once we move the cpu library to templates we plan on grouping the duplicated code anyway.